src/test/java/com/revature/util/ValidationUtilTest.java

```diff
@@ -0,0 +1,153 @@
+package com.revature.util;
+
+
+import java.util.ArrayList;
+import java.util.List;
+import static org.mockito.Mockito.*;
+import org.mockito.MockitoAnnotations;
+import org.mockito.Spy;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import com.revature.entities.Content;
+import com.revature.entities.Module;
+import com.revature.exceptions.InvalidContentException;
+import com.revature.exceptions.InvalidModuleException;
+import com.revature.exceptions.InvalidSearchException;
+import com.revature.testingutils.ContentFactory;
+import com.revature.util.ValidationUtil;
+
+public class ValidationUtilTest {	
+	String badString = ContentFactory.badS;
+	ValidationUtil vu;
+	Content content = null;
+	Module module = null;
+	@Spy ValidationUtil vuMock;
+	
+	
+	@BeforeTest
+	void setup() {
+		MockitoAnnotations.initMocks(this);
+		vu= new  ValidationUtil();
+	}
+	@BeforeMethod
+	void init() {
+		content = ContentFactory.getContent();
+		module = new Module();
+		
+	}
+	@Test(expectedExceptions = InvalidContentException.class)
+	public void verifyContent_TestContentEqualsNullThrowException() {
+		content=null;
+		vu.verifyContent(content);
+	}
+	@Test(expectedExceptions = InvalidContentException.class)
+	public void verifyContent_TestSecondCondition_MoreThanMAX_CHAR_LENGTH() {
+		content.setFormat(badString);
+		vu.verifyContent(content);
+	}
+	@Test(expectedExceptions = InvalidContentException.class)
+	public void verifyContent_TestSecondCondition_GetFormatIsEmpty() {
+		content.setFormat("");
+		vu.verifyContent(content);
+	}
+	
+	@Test(expectedExceptions = InvalidContentException.class)
+	public void verifyContent_TestThirdCondition_MoreThanMAX_CHAR_LENGTH() {
+		content.setTitle(badString);
+		vu.verifyContent(content);
+	}
+	@Test(expectedExceptions = InvalidContentException.class)
+	public void verifyContent_TestThirdCondition_GetTitleIsEmpty() {
+		content.setTitle("");
+		vu.verifyContent(content);
+	}
+	@Test(expectedExceptions = InvalidContentException.class)
+	public void verifyContent_TestFourthCondition_MoreThanMAX_CHAR_LENGTH() {
+		content.setUrl(badString);
+		vu.verifyContent(content);
+	}
+	@Test(expectedExceptions = InvalidContentException.class)
+	public void verifyContent_TestFourthCondition_GetUrlIsEmpty() {
+		content.setUrl("");
+		vu.verifyContent(content);
+	}
+	@Test(expectedExceptions = InvalidContentException.class)
+	public void verifyContent_TestFifthConditionThrowException() {
+		content.setDescription(badString);
+		vu.verifyContent(content);
+	}
```

Comment:
Basically, theese tests verify that the exception is trhown when there is an  Empty String, when a string has more  than 254 characters

Reply:
There still seem to be methods which neither have an assert method nor check for an exception. Please refactor these as they should not be included. Also, refactor the method signatures to following naming conventions. Thanks!

We can merge when this is complete